app/components/new-project-modal/template.hbs

```diff
@@ -80,12 +80,12 @@
             </div>
         {{/modal.body}}
         {{#modal.footer}}
-            <button onclick={{action closeModal}} class="btn btn-default">{{t 'general.cancel'}}</button>
+            <button onclick={{action closeModal false}} class="btn btn-default">{{t 'general.cancel'}}</button>
             <button onclick={{action create nodeTitle description searchSelected.id}} class="btn btn-success {{unless nodeTitle.length 'disabled'}}">{{t 'general.create'}}</button>
         {{/modal.footer}}
     {{else}}
         {{#modal.body}}
-            <button data-dismiss="modal" aria-label="Close" class="close" onclick={{action closeModal}}>
+            <button data-dismiss="modal" aria-label="Close" class="close" onclick={{action closeModal true}}>
```

Comment:
Shouldn't this be `false`?

Reply:
Don't think so, that's for when the project has been created. https://github.com/CenterForOpenScience/ember-osf-web/pull/106/files#diff-335e2a4a272a8e6f478ce06b233830c9R92

Reply:
oh, lol, I didn't see the else